schemas/2015-05-21-preview/Microsoft.DevTestLab.json

```diff
@@ -21,13 +21,13 @@
       },
       "required": [ "type", "apiVersion", "location" ]
     },
-    "environments": {
-      "description": "Required properties to create environment in an existing DevTest Lab",
+    "virtualMachines": {
+      "description": "Required properties to create a virtual machine in an existing DevTest Lab",
       "type": "object",
       "properties": {
         "type": {
           "enum": [
-            "Microsoft.DevTestLab/environments"
+            "Microsoft.DevTestLab/labs/virtualMachines"
```

Comment:
At least a valid JSON body for each resource type, negative tests are welcome as well.
Please check /tests folder for other RPs' tests.